views/pages/registration-form.ejs

```diff
@@ -58,28 +58,30 @@
                         </div>      
                         
                         <div class="form-group">
-                            <!-- Email -->
+                          <!-- Username --> 
                             <input class="input-field"
-                              type="email" 
-                              name="email" 
-                              id="email" 
+                              type="text" 
+                              name="userName" 
+                              id="username" 
                               index: true,
-                              placeholder="mmcfly@cusco.com"
-                              autocomplete="email"
+                              placeholder="Username" 
+                              autocomplete="username"
                               required
                             >
+
+                            >
                         </div>
 
                         <div class="form-group">
-                            <!-- Username --> 
-                              <input class="input-field"
-                                type="text" 
-                                name="userName" 
-                                id="username" 
-                                index: true,
-                                placeholder="Username" 
-                                autocomplete="username"
-                                required
+                          <!-- Email -->
+                          <input class="input-field"
+                            type="email" 
+                            name="email" 
+                            id="email" 
+                            index: true,
+                            placeholder="mmcfly@cusco.com"
```

Comment:
Can you change this placeholder to be `Email`?

Reply:
i totally forget but I am on it

Reply:
No Worries  😄